src/components/Section/Citizenship/CitizenshipConnector.jsx

```diff
@@ -74,6 +76,13 @@ const connectCitizenshipSection = (Component, {
       case 'Status':
         return {
           ...citizenship.Status,
+          usPassport: foreign.Passport,
+          validator: data => (
```

Comment:
I don't love this because IMO `connect` should only be used for passing data to a component from Redux. As an alternative I'd propose importing the Redux `state` directly into the CitizenshipValidator and looking at the value of `foreign.Passport` there, similar to how CitizenshipMultipleValidator looks at `application.Settings.formType` (this isn't a great pattern either but it's more in line with what we're doing in other places).